package.json

```diff
@@ -42,7 +42,8 @@
     "typescript": "^3.8.2"
   },
   "resolutions": {
-    "@types/vfile-message": "1.0.1"
+    "@types/vfile-message": "1.0.1",
+    "@types/react": "16.9.23"
```

Comment:
Multiple versions of `@types/react` in `node_modules` lead to a lot of suffering (global namespace conflicts).